<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QDialog" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1, 50, 32);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="credit_amount">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: rgb(126, 90, 148);
font: 14pt &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: white;
font: 14pt ;</string>
       </property>
       <property name="text">
        <string>Срок (мес.):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="period">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: rgb(126, 90, 148);
font: 14pt &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: white;
font: 14pt </string>
       </property>
       <property name="text">
        <string>Ставка (%):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="rate">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: rgb(126, 90, 148);
font: 14pt &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">color: white;
font: 14pt;</string>
       </property>
       <property name="text">
        <string>Тип ежемесячных &lt;br&gt;платежей:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="annuit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;
font: 14pt;</string>
         </property>
         <property name="text">
          <string>Аннуитетный</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="diff">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;
font: 14pt;</string>
         </property>
         <property name="text">
          <string>Дифференцированный</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color: white;
font: 14pt ;</string>
       </property>
       <property name="text">
        <string>Переплата по кредиту:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="overpayment">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: rgb(126, 90, 148);
font: 14pt &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color: white;
font: 14pt;</string>
       </property>
       <property name="text">
        <string>Общая выплата:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="total_payment">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: rgb(126, 90, 148);
font: 14pt &quot;Comic Sans MS&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QListWidget" name="monthly_payment">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: rgb(126, 90, 148);
font: 14pt &quot;Comic Sans MS&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color: white;
font: 14pt ;</string>
       </property>
       <property name="text">
        <string>Ежемесячный платеж:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: white;
font: 14pt;</string>
       </property>
       <property name="text">
        <string>Сумма кредита:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="calculate">
     <property name="styleSheet">
      <string notr="true">QPushButton {
border: 1px solid grey;
background-color: rgb(80, 220, 100);
border-radius: 2px;
color: white;
font: 14pt ;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
